docs/v3.x/concepts/templates.md

```diff
@@ -0,0 +1,79 @@
+# Templates
+
+A template is a pre-made Strapi configuration designed for a specific use case. It allows you to quickly boostrap a custom Strapi app.
+
+Here are some things a template may configure for you:
+
+- Collection types and single types
+- Components and dynamic zones
+- Plugins to install, or custom plugins
+
+::: warning
+Templates and starters are not the same thing:
+
+- A _template_ is a pre-made Strapi configuration
```

Comment:
Is it a configuration or a configured application?

Reply:
It's a configuration that, once merged using the CLI and its `--template` option, generates a configured application.

A template is not an application because it cannot be run on its own. It lacks many files, like database configs or the `package.json`. It's only useful once applied on top of a default Strapi app.

The wording is definitely tricky here, so let me know if I'm not clear

Reply:
I think it might be best to expand the description a bit as to how they differ

Reply:
I added more details in [this commit](https://github.com/strapi/strapi/pull/7626/commits/e327d8ccfdb1fed3c600ffb9245e089b5a2d7ddb)